topic-operator/src/main/java/io/strimzi/operator/topic/K8sTopicWatcher.java

```diff
@@ -91,6 +91,8 @@ public void eventReceived(Action action, KafkaTopic kafkaTopic) {
                     LOGGER.debugCr(logContext.toReconciliation(), "Ignoring {} to {} {} because metadata.generation==status.observedGeneration", action, kind, name);
                 }
             }
+        } else {
+            LOGGER.warnCr(logContext.toReconciliation(), "Topic {}/{} has no spec", kafkaTopic.getMetadata().getNamespace(), kafkaTopic.getMetadata().getName());
```

Comment:
shouldn't the reconciliation marker already bring namespace/name so we don't need it logged in the message?

Reply:
Oh let me check. I've seen sth similar used so added the same here. If it is redundant, I will get rid of it.

Reply:
Updated! Good catch, thanks!